Dockerfile

```diff
@@ -1,6 +1,24 @@
-FROM node:8.11.1
-RUN npm install http-server -g
+FROM node:10
 
-COPY ./build /
 EXPOSE 8000
-CMD ["http-server", "-p", "8000"]
+
+# install node http server
+RUN npm install -g http-server
+
+# Create directory for app
+RUN mkdir -p /jolt-wall
```

Comment:
This can be removed, I think.

> If the `WORKDIR` doesn’t exist, it will be created even if it’s not used in any subsequent Dockerfile instruction.

[Source](https://docs.docker.com/engine/reference/builder/#workdir)

Reply:
Ha, you are right. I did not know that. Fix coming up :)